src/fides/api/alembic/migrations/versions/7f7c2b098f5d_add_name_index_to_ctl_systems.py

```diff
@@ -0,0 +1,30 @@
+"""add indexes to ctl_systems and privacydeclaration
+
+Revision ID: 7f7c2b098f5d
+Revises: 1af6950f4625
+Create Date: 2023-11-21 18:52:34.508076
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "7f7c2b098f5d"
+down_revision = "1af6950f4625"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("CREATE EXTENSION IF NOT EXISTS pg_trgm;")
```

Comment:
The `pg_trgm` extension has been available since Postgres 8.3 so we should be ok to enable it